compiler+runtime/src/cpp/jank/read/parse.cpp

```diff
@@ -720,7 +720,7 @@ namespace jank::read::parse
     auto const sym(expect_object<obj::symbol>(sym_result.expect_ok().unwrap().ptr));
     auto const sym_end(sym_result.expect_ok().unwrap().end);
 
-    native_real n;
+    auto n = 0.0;
```

Comment:
Please use `native_real n{};`. *Always* use direct initialization. You could use `auto n{ 0.0 };` but you specifically want a `native_real` here.